.gitignore

```diff
@@ -51,3 +51,8 @@ bazel-*
 
 # VSCode
 .vscode
+
+# swagger generated source files
+sdk/python/kfp_*
+
+*.tar.gz
```

Comment:
This line needs to be removed since there are several tar.gz files checked in. e.g. https://github.com/kubeflow/pipelines/tree/master/backend/src/apiserver/server/test

Reply:
We could keep those checked in, but by default, this just wouldn't git add new .tar.gz files.  To me that seems like a good move, but what do you think?